### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by baezluise**
Form structure created and styled.

- Colors and borders added.
- Fonts added.
- Button actions added.

### CODE REVIEW

**Comment by baezluise** [comment]
-Centered the form.
-Added form image.
-Linked formspree.
-Deleted error in about me.

File: index.html [hunk 1]
```diff
@@ -151,5 +151,36 @@ <h2 class="card-name-2">Skills</h2>
     </div>
     </ul>
     </section>
+    <form class="contact-form" action="https://formspree.io/f/{mpzewvlb}" method="post">
+      <ul class="form-list">
+        <li class="para-form">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you work on the form action address by creating a form-spree account and generating an `id` to use in the form action in such a way that if a visitor sends a message to you from your portfolio it submits the form and you automatically receive the message in your mail. But currently, I could not reach out to you via the form-spree. Any time I tried to send a message it throw an error. Kindly follow this [link](https://formspree.io/html) for more understanding.

Error Message 👇 

hashid

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @baezluise, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baezluise** [comment]
Required changes made.

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @baezluise     👋 
 
 # STATUS: APPROVED ✔️  🍾  🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
- Linters are passing 
- Form submits successfully 

 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
 

 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**